### DISCUSSION TITLE
Adding OpenID Connect to custom login page

### DISCUSSION THREAD

**Discussion by Seul1018** [Budibase/budibase#14980] [Help]
Hi, so I tried making my own login template from this guide, [login template](https://github.com/Budibase/budibase/discussions/6098).
This guide basically shows how to handle login/logouts but does not explain implementing OpenID connect in the page. My question is, can I just add a button to redirect to the OIDC with a button. So a button that navigates to the OIDC with the config URL that I have already set up in my OpenID Connect setting.

Im also running Budibase version 2.8.1

![image](https://github.com/user-attachments/assets/4e017db1-8e7d-483a-8428-70cf67097243)

![image](https://github.com/user-attachments/assets/4e413126-97b0-4406-9008-5a32b8062aa8)

**Comment by joeDH00** [comment]
Im also working on my own custom login page and would like to know
